
 * check if cman_tool is installed

 * do not overwrite global return status 'rtrn' - use local keyword

Index: new/cman/init.d/cman.in
===================================================================
--- new.orig/cman/init.d/cman.in	2010-12-02 07:19:35.000000000 +0100
+++ new/cman/init.d/cman.in	2010-12-23 11:32:12.000000000 +0100
@@ -19,6 +19,8 @@
 # set secure PATH
 PATH="/bin:/usr/bin:/sbin:/usr/sbin:@SBINDIR@"
 
+test -x @SBINDIR@/cman_tool || exit 0
+
 chkconfig2()
 {
 	case "$1" in
@@ -46,7 +48,7 @@
 status()
 {
 	pid=$(pidof $1 2>/dev/null)
-	rtrn=$?
+	local rtrn=$?
 	if [ $rtrn -ne 0 ]; then
 		echo "$1 is stopped"
 	else
